aries_cloudagent/ledger/routes.py

```diff
@@ -740,6 +811,12 @@ async def register(app: web.Application):
             web.get(
                 "/ledger/multiple/get-write-ledger", get_write_ledger, allow_head=False
             ),
+            web.put("/ledger/multiple/{ledger_id}/set-write-ledger", set_write_ledger),
+            web.get(
+                "/ledger/multiple/get-write-ledgers",
+                get_write_ledgers,
+                allow_head=False,
+            ),
             web.get("/ledger/multiple/config", get_ledger_config, allow_head=False),
         ]
     )
```

Comment:
Do we really need the `/multiple/` path added? Whether the agent is connected to one ledger or more than one, I think the endpoints should still return information. The `ledger_id` can be used as path parameter where the operation requires one ledger instance to be identified.

Reply:
`/multiple/` has been removed. In case, multi-ledger config is not provided and single-ledger is applicable then the endpoint's response will return `ledger_id` as `default`. This is not applicable for `GET /ledger/config`, it will return a `Multiple ledger support not enabled` error.